src/nbl/video/CVulkanPhysicalDevice.h

```diff
@@ -2,27 +2,26 @@
 #define __NBL_C_VULKAN_PHYSICAL_DEVICE_H_INCLUDED__
 
 #include "nbl/video/IPhysicalDevice.h"
-
-#include <volk.h>
-
 #include "nbl/video/CVKLogicalDevice.h"
-#include "nbl/video/surface/ISurfaceVK.h"
+#include "nbl/video/surface/CSurfaceVulkan.h"
+#include "nbl/video/CVulkanCommon.h"
 
 namespace nbl::video
 {
         
 class CVulkanPhysicalDevice final : public IPhysicalDevice
 {
 public:
-    CVulkanPhysicalDevice(VkPhysicalDevice _vkphd, core::smart_refctd_ptr<system::ISystem>&& sys, core::smart_refctd_ptr<asset::IGLSLCompiler>&& glslc)
-        : IPhysicalDevice(std::move(sys), std::move(glslc)), m_vkphysdev(_vkphd)
+    CVulkanPhysicalDevice(VkPhysicalDevice vk_physicalDevice, core::smart_refctd_ptr<system::ISystem>&& sys,
+        core::smart_refctd_ptr<asset::IGLSLCompiler>&& glslc)
+        : IPhysicalDevice(std::move(sys), std::move(glslc)), m_vkPhysicalDevice(vk_physicalDevice)
     {
         // Get physical device's limits
         VkPhysicalDeviceSubgroupProperties subgroupProperties = { VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SUBGROUP_PROPERTIES };
         {
             VkPhysicalDeviceProperties2 deviceProperties = { VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_PROPERTIES_2 };
             deviceProperties.pNext = &subgroupProperties;
-            vkGetPhysicalDeviceProperties2(m_vkphysdev, &deviceProperties);
+            vkGetPhysicalDeviceProperties2(m_vkPhysicalDevice, &deviceProperties);
                     
             // TODO fill m_properties
```

Comment:
are there any more properties we can expose @achalpandeyy ?

Reply:
Just did the ray tracing ones for now as they're relevant to Erfan's work. I suggest to expose more as we need them.